code/src/components/Header.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import moment from 'moment';
```

Comment:
It would be nice if you separate imports of internal and external components with an empty line 😸